index.html

```diff
@@ -8,24 +8,29 @@
     <title>my-portfolio</title>
 </head>
 <body>
```

Comment:
Please do not merge PR without approval from a reviewer. KIndly reopen this PR and submit the active link to PR for review.

Reply:
It seems I can't reopen a merged pull request. Any guidance will be appreciated